client/my-sites/email/titan-setup-mailbox/index.js

```diff
@@ -60,22 +64,21 @@ const TitanSetupMailbox = ( { selectedDomainName } ) => {
 
 	return (
 		<>
-			<PageViewTracker
-				path={ analyticsPath }
-				title="Email Management > Set up your Professional Email"
-			/>
+			<PageViewTracker path={ analyticsPath } title="Email Management > Set Up Titan Mailbox" />
```

Comment:
I've renamed (and capitalized) the title to be more consistent with what we already have in other pages.

Reply:
Do we want to use "Titan" or "Professional Email" in the title? "Professional Email" starts getting verbose.

Reply:
Seems to me that the correct thing should be Professional Email. I think we are using now everywhere "Professional Email" and it might confuse the customer reading Titan instead of Professional Email. He might wonder at this point, what is Titan?

Reply:
I'm replying myself, this won't be visible to the user 🙂

Reply:
Yes, that particular string (which by the way is not localized on purpose) is only for Google Analytics. We already refer to `Titan` in the `Add New Mailboxes` component so I thought it would be a good idea to reuse the same term, which also has the benefit of being shorter 🙂.